Update the information for new meetings

abeltje opened the pull request:
No more COVID-19 restrictions, but still online meetings.

abeltje committed:
Update the information for new meetings

No more COVID-19 restrictions, but still online meetings.

waterkip commented on bin/announce_amsterdam_pm.pl:
```diff
@@ -75,22 +75,38 @@ sub send_email {
 
-Because of the corona regulations, the Amsterdam.pm meeting planned for tuesday %s at 20:00h will not take place,
-but we try to have a virtual one at %s
+After two years of COVID-19 restrictions, we all got used to online video meetings and Amsterdam.pm doesn't want to break that habit. The meeting planned for tuesday %s at 8PM will take place at https://meet.jit.si/amsterdam.pm
```
Nitpicking: hier breek je niet af op de 80 chars?

abeltje committed:
Git isa VCS, we don't need to do its work

Remove the previous version of the text, we don't need to keep that in
comments. If needed we can retreive from git.

abeltje committed:
Reformat English text to width 80
